src/Http/Controllers/Controller.php

```diff
@@ -53,9 +53,11 @@ public function insertUpdateData($request, $slug, $rows, $data)
             $content = $this->getContentBasedOnType($request, $slug, $row);
 
             // Set the content back to the previous value when there is really now input for this field
-            if (is_null($content) && is_null($request->input($row->field))) {
-                if (isset($data->{$row->field})) {
-                    $content = $data->{$row->field};
+            if (is_null($content)) {
+                if (is_null($request->input($row->field))) {
+                    if (isset($data->{$row->field})) {
```

Comment:
Code comments is really needed, I will take a good look at the code before we release the stable version `v1.0` and make sure that everything is commented and that all classes methods are minimal and splitted up into severals. It's on my todo for the `v1.0` release. Which I hope to come within a month.